src/shared/apis/queryCache.ts

```diff
@@ -44,8 +54,22 @@ export const hashCode = (rawText: string): string => {
   return `${hash}`
 }
 
+type CacheValue = {
+  dateSet: number
+  hashedVariables: string
+  isCustomTime: boolean
+  mutex: ReturnType<typeof RunQueryPromiseMutex>
+  status: RemoteDataState
+  error?: string
+  values?: RunQuerySuccessResult | null
+}
```

Comment:
As mentioned in the PR comments, I added `status` and `error` here in anticipation of future work. Both are being set in this PR but have no ramifications on the view layer